### PR TITLE
fix: adjust word order and enable animation on zh-hans

### PR DESCRIPTION
**Pull request by kecrily**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

adjust word order and enable animation on zh-hans

### CODE REVIEW

File: src/_data/sites/zh-hans.yml [hunk 1]
```diff
@@ -174,14 +174,14 @@ branding_page:
...
   
   animation:
-    enabled: false
-    text: 检测和修复 JavaScript 代码中的问题。
-    highlight_word: problems
-    wrong_word: problmes
+    enabled: true
+    text: 检测并修复 JavaScript 代码中的问题。
+    highlight_word: 代码中的问题。
+    wrong_word: 代码中的提问。
```
Comment (kecrily) [inline src/_data/sites/zh-hans.yml:184]: The reason why Chinese and Japanese cannot use animate is that CJK text does not use the same space segmentation as Western text.

Here `animate.text` is word-segmented into `检测并修复`, `JavaScript` and `代码中的问题。`. But actually is `检测`, `并修复`, `JavaScript`, `代码中的` and `问题`. So I used this wrong participle to expand the scope of `highlight_word`.

It's not a very nice solution, but it works. Another problem now is that the lint content cannot be changed, and pictures are used.

截屏2023-01-09 23 01 45

Comment (aladdin-add) [reply src/_data/sites/zh-hans.yml:184]: ideally the highlighted word should be "提问"(same as "problems"). just wondering if we can use an invisible zero-width char: "代码中的{zwc}问题"?

Comment (kecrily) [reply src/_data/sites/zh-hans.yml:184]: I tried zero-width spaces, but the tokenizer doesn't recognize it.

Comment (aladdin-add) [reply src/_data/sites/zh-hans.yml:184]: well, I think the current behaviour is fine.

**Review by aladdin-add — APPROVED** [review]
LGTM.

**Review by amareshsm — APPROVED** [review]
LGTM Thanks!

**Pull request merged**